pyp2spec/pyp2conf.py

```diff
@@ -46,12 +43,13 @@ def get_package_metadata(self, *,session=None):
         pkg_index = f"https://pypi.org/pypi/{self.package_name}/json"
         s = session or requests.Session()
         try:
-            response = s.get(pkg_index).json()
-        except json.decoder.JSONDecodeError:
+            response = s.get(pkg_index)
+            response.raise_for_status()
+        except requests.exceptions.HTTPError:
```

Comment:
This seems rather unnatural to me. The raise_for_status method is designed to propagate the exception further up. If you just want to catch it right away, wouldn't it be easier to use `if not response.ok:`?